tests/test_api.py

```diff
@@ -730,3 +732,50 @@ def test_delete_build_auth(testclient):
 
     # r = schema.APIResponse.parse_obj(response.json())
     # assert r.status == schema.APIStatus.ERROR
+
+
+def test_api_cancel_build(testclient):
```

Comment:
This test needs an unauth version to check that auth is working, see `test_delete_build_unauth`.